### PR TITLE
New route for contributors

### PR DESCRIPTION
**Pull request by mkalish**
This adds a new route to support showing a more detailed dive into the list of contributors for a particular candidate

## Description
- Added a new route
- Added the fixed-data-table react module
- Added a button that will navigate to this route for candidates race from the campaign compare page

## Motivation and Context
There is a desire to be able to do a deeper dive into campaigns and the actual contribtuors

## How Has This Been Tested?
Tested locally

## Screenshots (highly recommended):
screen shot 2016-05-28 at 6 33 15 pm

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] I've talked through the changes with a team lead
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by mkalish** [comment]
@romoy @jkoul

File: client/js/campaign/campaignScorecard.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (romoy) [inline client/js/campaign/campaignScorecard.jsx]: This file is in the old location.

File: client/js/campaign/campaignDetailComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```
Comment (romoy) [inline client/js/campaign/campaignDetailComponent.jsx]: this file is in the old location

File: client/css/fixed-data-table.min.css [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/**
```
Comment (romoy) [inline client/css/fixed-data-table.min.css]: this css is in the old location

File: client/js/campaign/campaignDetailComponent.jsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        const detailsPromise = Client.getCampaignData(id);
+        const scorePromise = Client.getCandidate({id: candidateId, campaigns: [{campaignId: id}]});
+        Promise.join(detailsPromise, scorePromise, function(details, score){
+            that.setState({contributions: details, campaignScore: score[0].campaigns[0], candidate: score[0].candidate});
+        });
+    }
+
+    render() {
+        const { contributions, campaignScore, candidate } = this.state;
+        console.log(campaignScore);
```
Comment (romoy) [inline client/js/campaign/campaignDetailComponent.jsx]: extra log message

File: client/js/campaign/campaignDetailComponent.jsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                            <Col xs={12}>
+                                <h2>{candidate.name}</h2>
+                            </Col>
+                        </Row>
+                    </Col>
+                    <Col xs={3}>
+                        <CampaignInfo info={campaignScore}></CampaignInfo>
+                    </Col>
+                    <Col xs={9}>
+                        <Table rowsCount={contributions.length}
```
Comment (romoy) [inline client/js/campaign/campaignDetailComponent.jsx]: can the table, and the cell width, and height values be responsive?

Comment (mkalish) [inline client/js/campaign/campaignDetailComponent.jsx]: What I can do is maybe fix the first three columns so they always show and the other ones are controlled by horizontal scroll bar.

**Comment by mkalish** [comment]
Closes #157

**Pull request merged**